scripts/benchmark-foundry.sh

```diff
@@ -1,3 +1,9 @@
 . ./.env
 
-time forge test -vv --fork-url $ETH_RPC_URL --fork-block-number $FORK_BLOCK --gas-limit $GAS_LIMIT
+cacheflag=""
+if [ $CLEAR_CACHE = 1 ]; then
+  rm -rf out
```

Comment:
The cache is in ~/.foundry/cache

Reply:
Yea, since the cache is stored there globally (instead of locally in the project directory) this approach seemed easier—this line clears the solidity cache, then `--no-storage-caching` avoids using the storage cache. I didn't want to delete the entire global cache folder, and conditionally deleting one of the cache folders based on chain + block is more tedious